src/en/contributing/http_api.md

```diff
@@ -0,0 +1,113 @@
+# Lemmy HTTP API
+
+
+<!-- toc -->
+
+- [Websocket vs HTTP API](#websocket-vs-http-api)
+- [Examples](#examples)
+  * [TypeScript](#typescript)
+  * [Curl](#curl)
+    + [GET](#get-example)
+    + [POST](#post-example)
+- [HTTP API exclusive features](#http-api-exclusive-features)
+  * [RSS/Atom feeds](#rss-atom-feeds)
+  * [Images](#images)
+    + [Create (request)](#create-request)
+    + [Create (response)](#create-response)
+    * [Delete](#delete)
+
+<!-- tocstop -->
+
+## WebSocket vs HTTP API
+Lemmy's HTTP API is almost identical to its WebSocket API:
```

Comment:
Make sure to add a newline after headers like the others, some markdown including commonmark need it. (not sure if mdbook's does but its best to be consistent).